Dockerfile

```diff
@@ -5,7 +5,7 @@ WORKDIR /app
 
 COPY dnd5esheets/front/package.json ./
 COPY dnd5esheets/front/package-lock.json ./
-RUN npm install
+RUN npm install --omit=dev
 COPY dnd5esheets/front/ ./
 RUN npm run build
```

Comment:
Did you test it? All the dependencies are dev dependencies. `npm run build` requires all the dev dependencies, and then we can discard the whole `front` and keep only `front/dist`.

Reply:
ah, in that case we can discard the PR 👍